### DISCUSSION TITLE
Fans set to 100% only reaching half their full speed

### DISCUSSION THREAD

**Discussion by bansama** [Rem0o/FanControl.Releases#1321] [Q&A]
Totally new to fancontrol. I've set up curves for the chassis fans to reach 100% at given temperatures to lesson the load on my GPU fan (the system is quieter when the chassis fans pull their weight). The problem is, with 100% I'm only getting 600 rpms on fans that should be doing over 1000. Armoury Crate lets me set fan speed manually based on four options, silent (off), standard (50%ish), turbo (75%ish), and full speed (100%ish). With turbo or full speed the chassis fans can spin at over 1000 rpm. But 100% in fancontrol is only reaching the equivalent of Armoury Crate's standard. 

Is there a way in fancontrol to specify the rpm range that 100% should be giving?

Motherboard is ASUS PRIME H570-PLUS. No idea what the fans are.

**Comment by Rem0o** [comment]
See https://github.com/Rem0o/FanControl.Releases/issues/1286

**Reply by bansama** [reply]
Guess I'll use Armoury Crate's fan options in tandem with Fancontrol. After a day of trying things, I've found that without Armoury Crate installed the chassis fans stick on running at 600+ rpms at 0% when they don't need to spin at all. And there's no other Asus software installed. Fan control in the bios is also set to manual. It's not ideal, but I'd rather not mess something up beyond repair so it'll do. Thanks.
